plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/linked-product-control.tsx

```diff
@@ -136,35 +147,40 @@ const LinkedProductControl = ( {
 	location: WooCommerceBlockLocation;
 	usesReference: string[] | undefined;
 } ) => {
-	const REFERENCE_TYPE_PRODUCT = 'product';
```

Comment:
I think it would be a good idea to keep the `const` instead of using the string ('product'). Using the const is less error prone. We could also create another constant for 'cart'.

Reply:
Makes sense to me. As we were using 'product' in only two places, therefore I removed the constant and replaced it with string. But Let's use constants for both 🤝